### ISSUE TITLE
Markdown not working

### ISSUE THREAD

**Issue by tubia** [sdelements/lets-chat#661]
Hi,
I have _let's chat_ installed and working but when I use markdown syntax in chat it's not displayed at all.
There is something to configure to make it works?

Thank you

**Comment by mmeinert** [comment]
you can try with #454, I personally didn't

**Comment by tubia** [comment]
Yes I saw that before but it's possible to merge pull requests in your local copy of the repo?
It's strange that this PR didn't get merged.

**Comment by funkaoshi** [comment]
I think there were performance issues with the change.
